utils/scripts/testconfig/test-variables.yml

```diff
@@ -2,46 +2,8 @@ BROWSER:
   NAME: BROWSER_NAME
   # List of Chrome options - https://peter.sh/experiments/chromium-command-line-switches/
   OPTIONS: add_argument("--ignore-certificate-errors");add_argument("window-size=1920,1024");add_argument('--headless');add_argument('--no-sandbox');add_argument('--disable-gpu')
```

Comment:
Why do we use headless? can't we run the test and see it running in the browser?